### ISSUE TITLE
[BUG] Abnormal execution time

### ISSUE THREAD

**Issue by jocelynke** [ChristianVisintin/ATtila#2]
**Describe the bug**
Very long execution time for at command AT+CSQ : about 3 seconds

**To Reproduce**
Run the following code : 
```python
atrunenv = ATRuntimeEnvironment(True)
atrunenv.configure_communicator(serial_port="/dev/ttyUSB2", baud_rate=9600, timeout=5)
atrunenv.open_serial()
response = atrunenv.exec("AT+CSQ;;OK;;0;;;;['AT+CSQ=?{rssi},','AT+CSQ=${rssi},?{ber}']")
print(response.full_response)
print(response.execution_time)
```

**Expected behavior**
An execution time lesser than 1 seconds

**Desktop (please complete the following information):**

- OS: Linux
- Architecture ARM
- Python version 3.7.4
- ATtila version 1.1.3

**Additional context**
The command used below (AT+CSQ) ran through a minicom returns instantly.
The same code with command "AT;;OK;;0" takes about 950ms, which is also quite high.

**Comment by veeso** [comment]
Hi,

thank you for reporting this issue.
Yes, the execution time was very slow on low baud rate and I managed to speed it up.
The current changes have been made in 'dev' branch.
I think the release will be published by the end of the day.

**Comment by jocelynke** [comment]
Great ! I'll try again with the next release

**Comment by veeso** [comment]
Fixed in commit c366fe5

ATtila 1.1.4 has been published.
Test it and let me know if you have other issues.

**Comment by jocelynke** [comment]
Fix confirmed ! Thanks
